x-pack/legacy/plugins/apm/server/lib/traces/get_trace_items.ts

```diff
@@ -35,7 +36,14 @@ export async function getTraceItems(
         bool: {
           filter: [
             { term: { [TRACE_ID]: traceId } },
-            { terms: { [PROCESSOR_EVENT]: ['span', 'transaction', 'error'] } },
+            {
+              terms: {
+                [PROCESSOR_EVENT]: [
+                  ProcessorEvent.span,
+                  ProcessorEvent.transaction
```

Comment:
Is this missing `error` or was this change intentional?

Reply:
oh, I'm sorry I missed this!